app/src/main/java/com/github/se/project/ui/components/LessonEditor.kt

```diff
@@ -118,7 +118,7 @@ fun LessonEditor(
     } else {
       onConfirm(
           Lesson(
-              lesson!!.id,
+              lesson?.id ?: "",
```

Comment:
The app does not crash anymore. The added lesson is uploaded on firestore but does not appear on the overview.